helm/cluster/values.yaml

```diff
@@ -1,27 +1,53 @@
 # Auto-generated by helm-values-gen, do not edit!
 
-connectivity:
-  network:
-    pods:
-      cidrBlocks:
-        - 100.64.0.0/12
-    services:
-      cidrBlocks:
-        - 172.31.0.0/16
+global:
+  components:
+    cri:
+      registries:
+        docker.io:
+          - endpoint: registry-1.docker.io
+          - endpoint: giantswarm.azurecr.io
+  connectivity:
+    network:
+      pods:
+        cidrBlocks:
+          - 100.64.0.0/12
+      services:
+        cidrBlocks:
+          - 172.31.0.0/16
+  controlPlane:
+    replicas: 3
+  metadata:
+    servicePriority: highest
 internal:
+  components:
+    cri:
+      sandboxContainerImage:
+        name: giantswarm/pause
+        registry: quay.io
+        tag: "3.9"
+  connectivity:
+    sshSsoPublicKey: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIM4cvZ01fLmO9cJbWUj7sfF+NhECgy+Cl0bazSrZX7sU vault-ca@vault.operations.giantswarm.io
   controlPlane:
+    kubeadmConfig:
```

Comment:
Will try to make this truly optional in JSON schema (allow `null`) so it's less noise for unused empty objects, but not very high priority.